Sources/NIO/Interfaces.swift

```diff
@@ -254,7 +254,7 @@ extension NIONetworkDevice {
                                                 dst: $0.baseAddress!,
                                                 size: INET_ADDRSTRLEN)
                 }
-                return SocketAddress(mask, host: mask.addressDescription())
+                return SocketAddress(mask)
```

Comment:
I may have missed something, why do we no longer care about the `host`?

Reply:
`addressDescription` is `internal` to NIOCore, and we added a new initializer to `SocketAddress` that defaults to this behaviour, so the call is identical.